src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -339,7 +352,7 @@ uct_rc_mlx5_txqp_inline_post(uct_rc_iface_t *iface, uct_rc_txqp_t *txqp, uct_ib_
 static UCS_F_ALWAYS_INLINE void
 uct_rc_mlx5_txqp_dptr_post(uct_rc_iface_t *iface, uct_rc_txqp_t *txqp, uct_ib_mlx5_txwq_t *txwq,
                            unsigned opcode_flags,
-                           const void *buffer, unsigned length, uint32_t *lkey_p,
+                           const uct_iov_t *iov, size_t iovcnt,
```

Comment:
@yosefe Should I delete these opcodes from original function? (currently I leave them all in original function)